v4/internal/awssupport/awssupport.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// +build go1.8
+
+package awssupport
```

Comment:
This package is required for the two sets of `aws-sdk` tests. All I did was copy the package from the v3 directory and change all instances of `v3` to `v4`.